From acdf9061fe5539c75604d2b7919cc434b2cebb02 Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@archlinux.org>
Date: Sun, 12 Aug 2018 23:14:44 -0400
Subject: [PATCH] Fix regression that broke installing desktop file

In commit 5d94db9c7940aee9af769ce314c1dd1be9ddc18c the desktop file was
moved from src/ to dist/ but the relative path from src/src.pro was
switched to an absolute path from the repository root. This broke
detection of the file from within qmake.

Fix by using the same $DIST_PATH used elsewhere for consistency, which
uses ../dist/.
---
 unixconf.pri | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/unixconf.pri b/unixconf.pri
index 3bb51975ff..164b835ef9 100644
--- a/unixconf.pri
+++ b/unixconf.pri
@@ -36,7 +36,7 @@ nogui:systemd {
 
 # Menu Icon
 !nogui {
-    menuicon.files = dist/unix/qbittorrent.desktop
+    menuicon.files = $$DIST_PATH/qbittorrent.desktop
     menuicon.path = $$DATADIR/applications/
     INSTALLS += menuicon
 
